services/git-service-impl/src/main/java/io/fabric8/launcher/service/github/impl/KohsukeGitHubServiceImpl.java

```diff
@@ -1,19 +1,5 @@
 package io.fabric8.launcher.service.github.impl;
 
-import java.io.FileNotFoundException;
```

Comment:
Why your import statements are ordered different than mine? Are you using the `fabric8-launcher` code style in https://github.com/fabric8-launcher/launcher-backend/blob/master/ide-configs/idea/fabric8-launcher.xml?

Reply:
I didn't. Default IDEA I believe. Discovered it after opening a PR, but now I am using the launcher one.